src/Exception/AuthException.php

```diff
@@ -8,14 +8,9 @@ class AuthException extends \Exception
 {
     public function __construct(
```

Comment:
The constructor? I suppose we can. We don't throw an instance of `AuthException` anywhere; it's just used as the base class for all other exceptions in this library. Those exceptions typically set their own messages. `ExceptionHandler` in equip/framework would interpret the `\Exception` default code of `0` as `500`, so we don't necessarily need to set that here. Want me to remove it?